### ISSUE TITLE
Custom variant => "unknown variant" error + autocompletion failure (but hover popup shows correct CSS)

### ISSUE THREAD

**Issue by danielweck** [tw-in-js/typescript-plugin#16]
In my Twind config:
```
variants: {
    'is-rtl': '&[dir=rtl]'
}
```

In my JSX:
```jsx
<p
dir="rtl"
class={`
    m-4
    p-2
    text-2xl
    border(
        2
        red-800
    )
    is-rtl:font-bold
    is-rtl:text-6xl
    is-rtl:border(
        4
        blue-800
    )
`}
>
اَلْعَرَبِيَّةُ Right To Left
</p>
```

Visual Studio Code:

<img width="582" alt="Screenshot 2021-03-23 at 22 22 13" src="https://user-images.githubusercontent.com/625201/112226675-5e14cf00-8c26-11eb-9f49-ad2cbccfd380.png">

Rendered HTML (with RTL, and with LTR to compare):

<img width="861" alt="Screenshot 2021-03-23 at 22 23 53" src="https://user-images.githubusercontent.com/625201/112226948-ce235500-8c26-11eb-9193-349e44c2938b.png">

<img width="861" alt="Screenshot 2021-03-23 at 22 24 13" src="https://user-images.githubusercontent.com/625201/112226967-d1b6dc00-8c26-11eb-853d-07416ddabfd5.png">

**Comment by sastan** [comment]
confirmed

**Comment by sastan** [comment]
Should be fixed with 0.3.4 and Twind IntelliSense 0.1.1

**Comment by danielweck** [comment]
Works great :) 👍
